homeassistant/components/econet/__init__.py

```diff
@@ -87,14 +88,11 @@ async def resubscribe(now):
         await hass.async_add_executor_job(api.unsubscribe)
         api.subscribe()
 
-    async def fetch_update(now):
-        """Fetch the latest changes from the API."""
+        # Refresh values
+        await asyncio.sleep(60)
         await api.refresh_equipment()
```

Comment:
Is this method updating all entities?
If yes, do we need the hourly polling in water_heater, as this method is also called every hour

Reply:
No, it doesn't update the water and energy usage. Those need to be requested separately, which is what the water heater does in its `async_update`.